hadoop-hdds/docs/content/tools/SCMCLI.md

```diff
@@ -1,27 +0,0 @@
----
```

Comment:
Could you please add a doc page for the new command?

Reply:
I deleted the old file because it didn't have any useful information. But I agree, it can be a good idea to create new documentation.

I created a short, initial version : I documented all the subcommand groups to make it clear what functionalities are available, but didn't specify all the details/flags as they can be get with `--help` easily.